docs/getting-started.md

```diff
@@ -158,9 +158,9 @@ please refer to this [guide](kind.md).
 ### Deploying Antrea in AKS, EKS, and GKE
 
 Antrea can work with cloud managed Kubernetes services, and can be deployed to
-AKS, EKS, and GKE clusters.
+AKS, AKS Engine, EKS, and GKE clusters.
```

Comment:
Does AKS Engine count as a "cloud managed Kubernetes service"?

Reply:
It is not. Would you remove "AKS Engine" here?

Reply:
I would yeah. Maybe keep this sentence the same and the text for the bullet list below the same as well (link could be changed to `aks-installation.md#deploy-antrea-to-an-aks-cluster`. Then add a new sentence after the list: `Antrea also supports AKS Engine (self-managed version of AKS) clusters, please refer this [installation guide](aks-installation.md#deploy-antrea-to-an-aks-engine-cluster)`. Would that work?

Reply:
I removed "AKS Engine" here. But did not change the bullet list item. Want to save some words here. I can change to what you suggested if you still prefer that.

Reply:
works for me